cumulus_library/module_allowlist.json

```diff
@@ -1,11 +1,13 @@
 {
     "desc" : 
         "this dict maps pip installable studies to expected paths inside site-packages",
-    "allowlist" : {
-        "covid_symptom" : "cumulus_library_covid",
-        "data_metrics" : "cumulus_library_data_metrics",
-        "rxnorm" : "cumulus_library_rxnorm",
-        "umls" : "cumulus_library_umls"
-
-    }
+    "allowlist" : [
```

Comment:
I added some new public studies (even though not all are on pypi yet) and removed the name - letting our manifest discovery code find the name (figured that would be more robust to name changes or whatever we do in future). But if you'd prefer to be explicit here, I can put them back no problem.

Reply:
nah this makes sense.